app/views/layouts/application.html.haml

```diff
@@ -22,12 +22,12 @@
       .wrappable
         .wrappable__content.wrappable__content--flex.main-pane
           %main
-            = render 'breadcrumbs'
+            = render 'application/breadcrumbs'
```

Comment:
Yes, an error of `ActionView::MissingTemplate` occurred if don't add `application`:

```
ActionView::Template::Error (Missing partial editor/posts/_breadcrumbs, editor/application/_breadcrumbs with {:locale=>[:ja], :formats=>[:html], :variants=>[], :handlers=>[:erb, :builder, :raw, :ruby, :haml]}. Searched in:
  * "/home/myokoym/work/sms/daimon-news-multi-tenant/app/views"
  * "/home/myokoym/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/kaminari-0.16.3/app/views"
  * "/home/myokoym/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/devise-3.5.6/app/views"
):
    22:       .wrappable
    23:         .wrappable__content.wrappable__content--flex.main-pane
    24:           %main
    25:             = render 'breadcrumbs'
    26:             = yield
    27:         %aside.wrappable__content.promotions
    28:           - if current_site.ad_client.present? && current_site.ad_slot.present?
  app/views/layouts/application.html.haml:25:in `_app_views_layouts_application_html_haml__3700888417442111477_69938080105520'
  app/controllers/editor/posts_controller.rb:56:in `preview'
```